demo/lightning/checkpoint/multinode/strategies.py

```diff
@@ -19,28 +21,49 @@
                                                        GCSDistributedWriter)
 
 
-def save_checkpoint_helper(rank, checkpoint, path, checkpoint_io, writer):
+def checkpoint_helper(checkpoint):
+    """Flatten the model and optimizer states into a new dict.
```

Comment:
Can you add a comment explaining why we need this ?

Reply:
Fixed the accuracy of this doc string.